### PR TITLE
python3-ansible-lint: update to 6.0.2.

### PR DESCRIPTION
**Pull request by jcgruenhage**
#### Testing the changes
- I tested the changes in this PR: **YES**

<del>Marked as draft because it's a prerelease still, but considering after creating the package locally for testing, I thought I might as well push it already so that it doesn't get lost.</del>

### CODE REVIEW

File: srcpkgs/python3-ansible-lint/template [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 wrksrc="${pkgname/python3-/}-${version}"
 build_style=python3-pep517
 make_install_target="ansible_lint-${version}-py3-none-any.whl"
 hostmakedepends="python3-wheel python3-toml"
-depends="ansible-core python3-yamllint python3-six python3-yaml python3-ruamel.yaml python3-tenacity python3-enrich python3-wcmatch"
+depends="ansible-core python3-yamllint python3-six python3-yaml python3-ruamel.yaml python3-tenacity python3-enrich python3-wcmatch python3-ansible-compat"
 short_desc="Linter for Ansible files"
 maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
-license="MIT"
+license="GPL-3.0-only"
```
Comment (jcgruenhage) [inline srcpkgs/python3-ansible-lint/template:12]: > The ansible-lint project is distributed as [GPLv3](https://github.com/ansible-community/ansible-lint/blob/main/COPYING) due to use of [GPLv3](https://github.com/ansible-community/ansible-lint/blob/main/COPYING) runtime dependencies, like ansible and yamllint.
> 
> For historical reasons, its own code-base remains licensed under a more liberal [MIT](https://github.com/ansible-community/ansible-lint/blob/main/docs/licenses/LICENSE.mit.txt) license and any contributions made are accepted as being made under original [MIT](https://github.com/ansible-community/ansible-lint/blob/main/docs/licenses/LICENSE.mit.txt) license.

File: srcpkgs/python3-subprocess-tee/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+make_install_target="subprocess_tee-${version}-py3-none-any.whl"
+hostmakedepends="python3-wheel python3-toml"
+depends="python3"
+short_desc="Replacement for subprocess.run capturing output while still printing it"
+maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license="MIT"
+homepage="https://github.com/pycontribs/subprocess-tee"
+distfiles="${PYPI_SITE}/s/${pkgname/python3-/}/${pkgname/python3-/}-${version}.tar.gz"
+checksum=ff5cced589a4b8ac973276ca1ba21bb6e3de600cde11a69947ff51f696efd577
+# requires molecule, which isn't packaged yet
```
Comment (jcgruenhage) [inline srcpkgs/python3-subprocess-tee/template:16]: I've given packaging `molecule` a shot multiple times, but I wasn't successful so far.

**Comment by ssbarnea** [comment]
Ansible-lint maintainer here, please keep this PR open until we make the official release of v6, maybe even one or two days later, as I suspect we might need some hotfixes. It would likely avoid extra work later.

BTW, I am really happy to see downstream being proactive about new releases.

File: srcpkgs/python3-ansible-lint/template (hunk 1 above)
Comment (paper42) [reply srcpkgs/python3-ansible-lint/template:12]: Could you leave this as a comment in the template, so if in the future someone is confused about this, they don't have to dig through all PRs?

File: srcpkgs/python3-ansible-lint/template [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
 pkgname=python3-ansible-lint
-version=5.4.0
+version=6.0.2
 revision=1
 wrksrc="${pkgname/python3-/}-${version}"
 build_style=python3-pep517
 make_install_target="ansible_lint-${version}-py3-none-any.whl"
 hostmakedepends="python3-wheel python3-toml"
-depends="ansible-core python3-yamllint python3-six python3-yaml python3-ruamel.yaml python3-tenacity python3-enrich python3-wcmatch"
+depends="ansible-core python3-yamllint python3-six python3-yaml python3-ruamel.yaml python3-tenacity python3-enrich python3-wcmatch python3-ansible-compat"
```
Comment (paper42) [inline srcpkgs/python3-ansible-lint/template:9]: wrap at 80 columns

File: srcpkgs/python3-ansible-lint/template (hunk 1 above)
Comment (jcgruenhage) [reply srcpkgs/python3-ansible-lint/template:19]: done

File: srcpkgs/python3-ansible-lint/template (hunk 3 above)
Comment (jcgruenhage) [reply srcpkgs/python3-ansible-lint/template]: done

**Pull request merged**